tests/virtual_machines_test.go

```diff
@@ -518,19 +518,35 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 
 			Context("and kubemacpool is opted-in on a namespace", func() {
+				var (
+					vm *kubevirtv1.VirtualMachine
+				)
 				BeforeEach(func() {
-					By("opting out the namespace")
```

Comment:
also in the opt out namespace, if the namespace wasn't opt out, kubemacpool should allocate the mac right?
Maybe this check, is needed as well?

Reply:
the opt-x wording is confusing, so let me clarify
in opt-in-mode - opting a namespace in, means it's "in the game", i.e. controlled by KMP

in other words, by opting in the namespace (as in the this BeforeEach), I expect that:
1. the mac will be allocated
2.  the mac in configmap will eventually get removed
so I think this is right. do you agree?

Reply:
Thanks for explaining. A lot clearer now. Fill free to ignore this note then. Sorry for the noise.

Reply:
no no, I think I'll try to make it clearer thanks to you.